### ISSUE TITLE
This repo is outdated and should be removed in favor of new example location

### ISSUE THREAD

**Issue by denisoby** [davidkpiano/xstate-todomvc#2]
Tried using this example and it does not - text is not modified, action "assign" does not work correctly.

However, found that up-to-date working example is located in another repo
https://github.com/statelyai/xstate/tree/main/examples/todo-mvc-react

I think https://github.com/davidkpiano/xstate-todomvc repo should be removed to avoid confusion, or should redirect to a new location.
Thank you!

**Comment by infinite-persistence** [comment]
Yeah, came here from https://xstate.js.org/docs/packages/xstate-react/#examples, and pretty much wasted time on outdated stuff.
